src/calculate_bucket.hpp

```diff
@@ -77,7 +77,9 @@ class F1Calculator {
     inline F1Calculator(uint8_t k, const uint8_t* orig_key)
     {
         uint8_t enc_key[32];
+        size_t buf_blocks = CDIV((k + kExtraBits) << kBatchSizes, kF1BlockSizeBits) + 2;
```

Comment:
note that my patch of making this a function (and spell lower case `cdiv()`) has landed in `master` already. So this would need to change before landing.

You may want to rebase the patch on top of master to catch any other potential issues like this